src/frontend/src/views/Data/index.tsx

```diff
@@ -95,19 +111,24 @@ const Data: FunctionComponent = () => {
       return;
     }
 
-    const transformedData = category.data.map((datum: BioinformaticsData) => {
-      const transformedDatum = Object.assign({}, datum);
-
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion -- Asserted above
-      category.transforms!.forEach((transform) => {
-        const methodInputs = transform.inputs.map((key) => datum[key]);
-        transformedDatum[transform.key] = transform.method(methodInputs);
-      });
-
-      return transformedDatum;
-    });
+    const transformedData = map(
```

Comment:
Just curious, what does the lodash map get us over the native Array `.map` here?

Reply:
same as above